README.md

```diff
@@ -85,7 +85,8 @@ Login and Password to token**
 > credentials.
 
```

Comment:
By trial and error I found that the "Secret text" token, at least for a public github.com account, should contain just the token value ("ghp_Vari0usChars"); the "Test Connection" button helps see if that works. It seems that github knows the user name from that token alone.

I did not uncover what "in corresponding domain" here means in practice, and "Credentials are filtered on changing custom GitHub URL" in help text. My credential ID and description deliberately did not mention neither the account name nor github.com, and it still worked for the default https://api.github.com URL.

This would be nice to clarify in the doc and help, while we are at it.

Reply:
I adding some some text and screen shorts would be nice, I think.